packages/syft/src/syft/service/user/user.py

```diff
@@ -106,6 +108,7 @@ def salt_and_hash_password(password: str, rounds: int) -> tuple[str, str]:
     return salt_bytes, hashed_bytes
 
 
+@cache
```

Comment:
Is the caching only to speed up CI? Not sure we want to do the caching with real passwords

Reply:
moved caching to conftest